samples/net/coap_client/src/coap-client.c

```diff
@@ -174,28 +214,17 @@ static void event_iface_up(struct net_mgmt_event_callback *cb,
 	struct net_pkt *pkt;
 	struct net_buf *frag;
 	int r;
-	u8_t observe = 0;
 
-	r = net_context_get(PF_INET6, SOCK_DGRAM, IPPROTO_UDP, &context);
-	if (r) {
-		printk("Could not get an UDP context\n");
-		return;
-	}
+	u8_t My_COAP_token1[8] = "puttok";
```

Comment:
Is there a reason why the length is specified here, for example token1 is only 7 chars long. Could you omit the length?